src/c4/Timer.hh

```diff
@@ -344,7 +336,9 @@ double Timer::system_cpu() const {
 double Timer::user_cpu() const {
   Require(!timer_on);
 #if defined(WIN32)
-  return difftime(tms_end, tms_begin);
+  using namespace std::chrono;
+  duration<double> diff = tms_end - tms_begin;
+  return duration_cast<nanoseconds>(diff).count() / 1.0e9;
 #else
   return (tms_end.tms_utime - tms_begin.tms_utime) /
```

Comment:
I don't have a lot of experience with `duration_cast` but I'm guessing that it can't be any more accurate that the system call provided by `tms_utime` since they are all based on the cpu cycle time.